LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 bitshares-fc
```

Comment:
That doesn't make sense, bitshares-fc is not an entity that can own copyright.

AFAIK the only use-case for the copyright owner is when it comes to defending the license in court. Which is unlikely to happen, given the liberal nature of the MIT license as such.

We could use "The BitShare Core Development Team" (although that's not really a legally existing entity either), or perhaps the BBF. @xeroc  @ryanRfox opinions?

Reply:
Don't think it has to be a legal entity at all.

"BitShares Core Development Team" or "BitShares Developers" would be fine I think.

Reply:
I suggest "Copyright (c) 2015-2018 Daniel Larimer and contributors."